test/serde/serde_helpers.py

```diff
@@ -1955,3 +1955,26 @@ def compare(detailed, original):
             "cmp_detailed": compare,
         }
     ]
+
+
+def make_paillier(**kwargs):
```

Comment:
If it won't block this PR, we could create an issue and kick this part over to the Crypto team.

Reply:
I'll create an issue after the merge.